### ISSUE TITLE
Show partially hidden email

### ISSUE THREAD

**Issue by udaynwa** [the-deep/client#2424]
**Insert Screenshot**
![image (2).png](https://zube.io/files/deep/e2e39e6d341f9521b5f5adda0f12c849-image-2.png)![image.png](https://zube.io/files/deep/67ddeb4cb25bdc7bf8d2f94ef6dd535a-image.png)![image (1).png](https://zube.io/files/deep/9f3ba90849e61d2eac03d4bcbda281e2-image-1.png)

## Acceptance Criteria
Show partially hidden email (https://github.com/the-deep/server/pull/1068), next to user's name in user select dropdown.

It should primarily be present in user add dropdown (in project panel > users), but also in other comment assignee and source assignee.

Basically need to modify UserSearchSelectInput and UserMultiSearchSelectInput.
